recipe/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+if [[ $target_platform == linux-aarch64 ]]; then
+    cmake   -DCMAKE_INSTALL_PREFIX=$PREFIX \
+            -DCMAKE_CXX_COMPILER=clang++ \
+    ..
```

Comment:
```suggestion
    cmake .. ${CMAKE_ARGS} \
          -GNinja \
          -DCMAKE_PREFIX_PATH=${PREFIX} \
          -DCMAKE_INSTALL_PREFIX=${PREFIX} \
          -DCMAKE_BUILD_TYPE=Release \
          -DCMAKE_CXX_COMPILER=clang++
```

Reply:
We don't need to set `CMAKE_CXX_COMPILER`.

Reply:
the -DCMAKE_PREFIX_PATH=${PREFIX} 
giving me such error:

CMake Warning:
  No source or binary directory provided.  Both will be assumed to be the
  same as the current working directory, but note that this warning will
  become a fatal error in future CMake releases.


CMake Error: The source directory "/home/mwaszkiewicz/miniconda3/conda-bld/mcl_1702977335269/work/build" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.